src/resonanceReconstruction/rmatrix/legacy/unresolved/CompoundSystem/test/evaluate.test.hpp

```diff
@@ -0,0 +1,1114 @@
+      CHECK( 7.3617143134432972 == Approx( xs[ elas ].value ) );
+      CHECK( 1.5670838905457892E-002 == Approx( xs[ capt ].value ) );
+      xs.clear();
+
+      system.evaluate( 6e+4 * electronVolt, xs );
+      CHECK( 2 == xs.size() );
+      CHECK( 7.1763623147011355 == Approx( xs[ elas ].value ) );
+      CHECK( 1.3406527410576555E-002 == Approx( xs[ capt ].value ) );
+      xs.clear();
+
+      system.evaluate( 7e+4 * electronVolt, xs );
+      CHECK( 2 == xs.size() );
+      CHECK( 7.0369090178438851 == Approx( xs[ elas ].value ) );
+      CHECK( 1.1714340063536310E-002 == Approx( xs[ capt ].value ) );
+      xs.clear();
+
+      system.evaluate( 8e+4 * electronVolt, xs );
+      CHECK( 2 == xs.size() );
+      CHECK( 6.9279078516440951 == Approx( xs[ elas ].value ) );
+      CHECK( 1.0400317466079910E-002 == Approx( xs[ capt ].value ) );
+      xs.clear();
+
+      system.evaluate( 9e+4 * electronVolt, xs );
+      CHECK( 2 == xs.size() );
+      CHECK( 6.8402239876247961 == Approx( xs[ elas ].value ) );
+      CHECK( 9.3501533213065698E-003 == Approx( xs[ capt ].value ) );
+      xs.clear();
+
+      system.evaluate( 1e+5 * electronVolt, xs );
+      CHECK( 2 == xs.size() );
+      CHECK( 6.7680817547271817 == Approx( xs[ elas ].value ) );
+      CHECK( 8.4916589760332239E-003 == Approx( xs[ capt ].value ) );
+      xs.clear();
+    } // THEN
+  } // GIVEN
+
+  GIVEN( "Pu239 unresolved resonance data" ) {
+
+    // test based on Pu239 ENDF/B-VIII.0 LRU=2 unresolved resonance evaluation
+    // cross section values extracted from NJOY2016.57
+
+    // this test is essentially for full energy dependent parameters without
+    // competition
+
+    // scattering radius from equation D.14
+    double a = 0.123 * std::pow( 236.9986 * 1.008664, 1. / 3. ) + 0.08;
+
+    // particles
+    Particle neutron( ParticleID( "n" ), neutronMass, 0.0 * coulombs, 0.5, +1);
+    Particle pu239( ParticleID( "Pu239" ), 236.9986 * neutronMass,
+                    94.0 * coulombs, 0.5, +1); // unknown parity
+
+    // particle pairs
+    ParticlePair in( neutron, pu239 );
+
+    // channels
+    Channel< Neutron > elastic00( in, in, 0. * electronVolt, { 0, 0.5, 0.0, +1 }, { a * rootBarn, 0.946 * rootBarn } );
+    Channel< Neutron > elastic01( in, in, 0. * electronVolt, { 0, 0.5, 1.0, +1 }, { a * rootBarn, 0.946 * rootBarn } );
+    Channel< Neutron > elastic10( in, in, 0. * electronVolt, { 1, 0.5, 0.0, +1 }, { a * rootBarn, 0.946 * rootBarn } );
+    Channel< Neutron > elastic11( in, in, 0. * electronVolt, { 1, 0.5, 1.0, +1 }, { a * rootBarn, 0.946 * rootBarn } );
+    Channel< Neutron > elastic12( in, in, 0. * electronVolt, { 1, 0.5, 2.0, +1 }, { a * rootBarn, 0.946 * rootBarn } );
+
+    // corresponding unresolved resonance tables
+    ResonanceTable table00(
+      { Resonance( 2.500000e+3 * electronVolt, 8.917200e+0 * electronVolt, 9.508100e-4 * rootElectronVolt, 4.070000e-2 * electronVolt, 2.842000e+0 * electronVolt, 0. * electronVolt ),
```

Comment:
These lines are much too long to fit the style guid. You could call your variable `eV` instead of `electronVolt` and save a lot of space.